### DISCUSSION TITLE
feature: Items that disappear from inventory when equipped

### DISCUSSION THREAD

**Discussion by Joak1ngglb** [AscensionGameDev/Intersect-Engine#2403] [Ideas]
### Description

It's simple, I think that nowadays it's a feature that every MMO should have. Every time an item is equipped it should disappear from the inventory and appear equipped in the character panel. Currently, when an item is equipped, it appears both in the inventory and in the character panel.

### Intended Use-Case

With this feature we could provide a more realistic system regarding equipment. It makes no sense for the item to be duplicated by appearing in the inventory and in the character panel.
I'm going to show a review they made of one of the games made with the engine, a game that was recently released on Steam.

![Sin título](https://github.com/user-attachments/assets/35df9f20-3fb7-4b1b-beab-4b01f9c8d756)

### Duplicate Check

- [X] This feature request is not a duplicate to the best of my knowledge.

**Comment by Joak1ngglb** [comment]
Sorry, I think I got the wrong section, it should be an enhancement

**Comment by Arufonsu** [comment]
dupe of  [Equipped items shouldn't use up Inventory Slots. #1493](https://github.com/AscensionGameDev/Intersect-Engine/discussions/1493)
